manifest.yml

```diff
@@ -0,0 +1,24 @@
+---
+## PWS information section
```

Comment:
doest this file needs to be named `manifest.yml` or is that configurable? Please give more elaboration on this file an it's usage. People should not change (but copy) files that are under source control. So please rename this file with something with example or move to docs folder.

Reply:
It can be named anything as the filename can be passed to the deploy command of the cloud foundry CLI.  Putting it at the root level with the default name makes the resulting command easier on the user. I figured that since everyone who uses this has their own fork, they could just edit that file in their own fork.

Reply:
Correct. A `cf push` from within the root directory to PWS will find and use a `manifest.yml` file if it is named as such. If named differently, you have to explicitly point at it. So having a default `manifest.yml` file at root is an accelerator to the PWS user to spare them from having to load in environment variables (and tune the app with mem and disk settings) by hand later.

Reply:
@forddonald @thadc23 I think that would be better. I don't like people to put credentials in files, because history has proven that they will end up in SCM systems and will be made publicly available. Given the fact that Nightscout saves personal information we should try to prevent that.